.github/labeler.yml

```diff
@@ -1,4 +1,4 @@
 maintenance:
-- .github/**/*
-- .pre-commit-config.yaml
-- VERSION
+- any-glob-to-any-file: .github/**/*
+- any-glob-to-any-file: .pre-commit-config.yaml
+- any-glob-to-any-file: VERSION
```

Comment:
You could just do:
```suggestion
- any-glob-to-any-file: ['.github/**/*', '.pre-commit-config.yaml', 'VERSION']
```

Reply:
That is so true. My first intent was to show people that you don't necessary need the list.
I will approve. Thanks @germa89